src/Containers/SessionChoice/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Col, Row } from 'antd';
+import React, { useState } from 'react';
+
+import Button from 'Components/Button';
+import Input from 'Components/Input';
+import { Box, CenteredContainer, OptionalText, Tags } from 'Components/CommonComponents';
+import pushNotification from 'pushNotification';
+
+interface Payload {
+  nickname: string;
+  game_code: string;
+}
+
+interface Response {
+  player_id: number;
+  game_code: number;
+  token: string;
+  error: string;
```

Comment:
`error` cannot be a string type field in the interface. Check how we did it [here](https://github.com/AverageDanteEnjoyer/virtual-clinic-web/blob/main/src/Containers/EditProfileForm/index.tsx#L61)